docs/guides/src/main/server/db.adoc

```diff
@@ -5,7 +5,7 @@
 <@tmpl.guide
     title="Configuring the database"
     summary="An overview about how to configure relational databases"
-    includedOptions="db db-*">
+    includedOptions="db db-* transactions-xa-enabled">
```

Comment:
You changed the category name to be in the singular but the option is still in the plurar. To keep consistency, please use the singular version.

Reply:
Thanks for noticing, changed in all places :)